File: 0455-assign-cookies/0455-assign-cookies.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);
        int i=0;
        for(int j=0;j<s.length;j++){
            if(i<g.length  && g[i]<=s[j] ){
                i++;
            }
        }
        
        return i;
    }
}

File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        ans=0
        i=0
        for num in s:
            if i<len(g) and g[i]<=num:
                i+=1
        return i
            
        